python/processors/aptos_ambassador_token/processor.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import grpc
+import ast
+import json
+
+from aptos.indexer.v1 import raw_data_pb2_grpc
+from aptos.indexer.v1 import raw_data_pb2
+from aptos.transaction.testing1.v1 import transaction_pb2
+from utils.config import Config
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-c", "--config", help="Path to config file", required=True)
+args = parser.parse_args()
+config = Config.from_yaml_file(args.config)
+starting_version = 0
+if config.starting_version_default != None:
+    starting_version = config.starting_version_default
+if config.starting_version_override != None:
+    starting_version = config.starting_version_override
+metadata = (
+    ("x-aptos-data-authorization", config.indexer_api_key),
+    ("x-aptos-request-name", "ambassador token indexer"),
+)
+options = [("grpc.max_receive_message_length", -1)]
+
+
+with grpc.insecure_channel(config.indexer_endpoint, options=options) as channel:
+    stub = raw_data_pb2_grpc.RawDataStub(channel)
+    current_transaction_version = starting_version
+
+    for response in stub.GetTransactions(
+        raw_data_pb2.GetTransactionsRequest(starting_version=starting_version),
+        metadata=metadata,
+    ):
+        assert response.chain_id == config.chain_id
+        for transaction in response.transactions:
+            assert transaction.version == current_transaction_version
+
+            found = False
+            if transaction.type == transaction_pb2.Transaction.TRANSACTION_TYPE_USER:
+                for event in transaction.user.events:
+                    addr = "0x" + event.key.account_address
```

Comment:
For standardizing addresses, we should always use this [helper function](https://github.com/aptos-labs/aptos-indexer-processors/pull/9/files#diff-fb72abdf04c829b4d8b5a8108bd0b604f4c63adefe683dd3b154098e596eee90R15). I haven't merged it yet but feel free to add it in this PR.